src/Routes.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Main from './pages/Main/Main';
+import List from './pages/List/List';
 
 class Routes extends React.Component {
   render() {
     return (
       <Router>
         <Switch>
           <Route exact path="/" component={Main} />
+          <Route exact path="/List" component={List} />
+          <Route exact path="/List:search" component={List} />
```

Comment:
- 라우트 경로는 소문자로 작성해 주세요!
```suggestion
          <Route exact path="/list" component={List} />
          <Route exact path="/ist:search" component={List} />
```